pontoon/base/static/js/fluent_interface.js

```diff
@@ -373,15 +373,25 @@ var Pontoon = (function (my) {
       /*
        * Is element of type that can be presented as a simple string:
        * - TextElement
-       * - ExternalArgument
-       * - MessageReference
+       * - Placeable with expression type ExternalArgument or MessageReference
        */
       isSimpleElement: function (element) {
-        return [
-          'TextElement',
-          'ExternalArgument',
-          'MessageReference'
-        ].indexOf(element.type) >= 0;
+        if (element.type === 'TextElement') {
+          return true;
+        }
+
+        // Placeable
+        if (
+          element.expression &&
+          [
+            'ExternalArgument',
+            'MessageReference'
```

Comment:
Perhaps it would be useful to factor this out to a `isSimplePlaceable` function which would check for any of the following [inline expression types](https://github.com/projectfluent/fluent/blob/69ecabcc06cf6f5b3654c439a9defc8575744313/spec/fluent.asdl#L46-L52):

    StringExpression(string)
    NumberExpression(number)
    MessageReference(iden id)
    ExternalArgument(iden id)
    AttributeExpression(iden id, iden name)
    VariantExpression(iden id, varkey key)
    CallExpression(fun callee, arg* arguments)

Reply:
Oh, I see, this is about expressions supported in the rich editor right now. Never mind then.